test/browser/commands.test.js

```diff
@@ -1,87 +1,131 @@
-import { clickAt, clickElem, expect, fixture, focusElem, hoverAt, hoverElem, sendKeys, sendKeysElem, setViewport } from '../../src/browser/index.js';
+import { clickAt, clickElem, dragDropElems, expect, fixture, focusElem, hoverAt, hoverElem, sendKeys, sendKeysElem, setViewport } from '../../src/browser/index.js';
 import { html } from 'lit';
 import { spy } from 'sinon';
 
 describe('commands', () => {
 
-	let elem;
-	const input = html`<input type="text">`;
```

Comment:
I needed a more complex fixture template for drag & drop, so I refactored the tests here so that click/hover tests use a button, keyboard tests use this input and the drag & drop has its own. The tests themselves didn't change.